packages/tags/stories/examples/CustomHue.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { Tag } from '@zendeskgarden/react-tags';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+
+interface ICustomHueStoryProps {
+  hue: string;
+}
+
+export const CustomHue: Story<ICustomHueStoryProps> = ({ hue }) => (
```

Comment:
This story no longer demonstrates the previous Styleguidist example: https://zendeskgarden.github.io/react-components/tags/#!/Advanced/3

Reply:
This example includes the single hue color picker.

![image](https://user-images.githubusercontent.com/4030377/105106213-7b80bd80-5a6a-11eb-9434-8f640aa0067d.png)

What other controls are missing?

Reply:
My bad.